safe/impact_function/impact_function.py

```diff
@@ -1704,7 +1705,7 @@ def provenance(self):
         self._provenance['analysis_extent'] = (
             self.analysis_extent.exportToWkt()
         )
-        self._provenance['data_store_uri'] = self.datastore.uri
+        self._provenance['data_store_uri'] = self.datastore.uri.absolutePath()
```

Comment:
You shouldn't call absolutePath on the URI, because I hope a the short future, we will be able to provide the postgis datastore. So absolutePath will not exist on this URI. I made a fix in PR #3685